src/components/CustomerQuota/NoQuota/NoQuotaCard.tsx

```diff
@@ -221,8 +221,18 @@ export const NoQuotaCard: FunctionComponent<NoQuotaCard> = ({
   const showGlobalQuota =
     !!quotaResponse?.globalQuota &&
     cart.length > 0 &&
+    /**
+     * We only display global limit messages if there is only one global limit,
+     * since we have not catered for showing global limit messages for multiple
+     * categories.
+     */
+    quotaResponse.globalQuota.length === 1 &&
     !!getProduct(cart[0].category)?.quantity.usage;
```

Comment:
Same question here, will the first cart item always be the same policy as what is in globalQuota?

Reply:
Same as above, will change this check to just check `allProducts` length.